src/sparseml/pytorch/torchvision/train.py

```diff
@@ -537,6 +547,43 @@ def collate_fn(batch):
     print(f"Training time {total_time_str}")
 
 
+def _create_model(
+    arch_key: str,
+    num_classes: int,
+    checkpoint_path: Optional[str],
+    recipe_path: Optional[str],
+    pretrained: bool,
+    pretrained_dataset: Optional[str],
+):
+    if checkpoint_path and checkpoint_path.startswith("zoo"):
```

Comment:
the zoo logic already happens with `ModelRegistry.create`, no? why duplicate here?

Reply:
Oh I didn't realize that - this was copied from the image_classification stuff, I can just remove this then